ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -307,6 +307,28 @@ process_change_point
+    mkdir -p "$EXP_DIR"
+    EXP_BINARY="/tmp/orion-original-edivisive"
+
+    echo "Downloading experimental orion binary..."
+    if ! curl -fsSL "https://github.com/cloud-bulldozer/orion/releases/download/orig-edivisive-exp/orion-amd64" -o "$EXP_BINARY"; then
+        echo "Failed to download experimental orion binary, skipping."
+        exit 0
+    fi
+    chmod +x "$EXP_BINARY"
+
+    echo "Running experimental orion (original e-divisive)..."
+    "$EXP_BINARY" --node-count ${IGNORE_JOB_ITERATIONS} --config ${ORION_CONFIG} ${EXTRA_FLAGS} --viz | tee "$EXP_DIR/${FILENAME}.txt" || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)